suite-native/module-home/src/screens/HomeScreen/components/PortfolioGraph.tsx

```diff
@@ -56,8 +50,6 @@ export const PortfolioGraph = forwardRef<PortfolioGraphRef>((_props, ref) => {
         [refetch],
     );
 
-    if (isDeviceDiscoveryEmpty && isDiscoveryActive) return null;
```

Comment:
shouldnt this be the code? 

`if (isDeviceDiscoveryEmpty && isDiscoveryActive) return null;`

So we dont show graph for when there is a discovery or therea are no accounts after it is done

Reply:
Meaning that everything else would revert to original code in this case

Reply:
Now I read that this is covered somewhere else though